tests/drivers/sensor/sbs_gauge/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+set(CONF_FILE "proj.conf")
```

Comment:
Just name the file prj.conf; the build system will find it by default.

Reply:
removed the line 5 with the proj.conf